geopandas/_compat.py

```diff
@@ -137,16 +137,21 @@ def set_use_pygeos(val=None):
     if USE_PYGEOS and env_use_pygeos is None and SHAPELY_GE_20:
         warnings.warn(
             "Shapely 2.0 is installed, but because PyGEOS is also installed, "
-            "GeoPandas will still use PyGEOS by default for now. To force to use and "
-            "test Shapely 2.0, you have to set the environment variable USE_PYGEOS=0. "
+            "GeoPandas still uses PyGEOS by default. However, starting with the next "
+            "release (0.14), the default will switch to Shapely. To already force to "
```

Comment:
Suggestion: `starting with the next release (0.14)` => `starting with version 0.14`?